tests/api/openapi.yaml

```
@@ -566,6 +570,12 @@ paths:
           required: true
           schema:
             type: string
+        - name: createDirectory
+          in: query
+          description: Indicate if we want to create a directory depending in the path given
```

```suggestion
          description: Indicate if we want to create the directory on the file system
```

Just to be clear about the directory on file system, versus a "Lizmap repository".